change-camera-controller/src/main/java/com/esri/arcgismaps/sample/changecameracontroller/MainActivity.kt

```diff
@@ -0,0 +1,252 @@
+        DataBindingUtil.setContentView(this, R.layout.activity_main)
+    }
+
+    private val sceneView by lazy {
+        activityMainBinding.sceneView
+    }
+
+    // custom toolbar to show camera controller options menu
+    private val toolbar by lazy {
+        activityMainBinding.toolbar.apply {
+            // set the overflow icon to the camera icon
+            overflowIcon = ContextCompat.getDrawable(context, R.drawable.ic_camera)
+        }
+    }
+
+    // graphics overlay for the scene to draw the 3d graphics on
+    private val graphicsOverlay by lazy {
+        GraphicsOverlay().apply {
+            // set the altitude values in the scene to be absolute
+            sceneProperties.surfacePlacement = SurfacePlacement.Absolute
+        }
+    }
+
+    // list of available asset files
+    private val assetFiles by lazy {
+        resources.getStringArray(R.array.asset_files).toList()
+    }
+
+    // the graphic representing the airplane 3d model
+    private val airplane3DGraphic by lazy {
+        // location for the target graphic
+        val point = Point(-109.937516, 38.456714, 5000.0, SpatialReference.wgs84())
+        // create the graphic with the target location
+        Graphic(point)
+    }
+
+    // camera controller which orbits a target location
+    private val orbitLocationCameraController by lazy {
+        // target location for the camera controller
+        val point = Point(-109.929589, 38.437304, 1700.0, SpatialReference.wgs84())
+        // instantiate a new camera controller with a distance from the target
+        OrbitLocationCameraController(point, 5000.0).apply {
+            // set a relative pitch to the target
+            setCameraPitchOffset(3.0)
+            // set a relative heading to the target
+            setCameraHeadingOffset(150.0)
+        }
+    }
+
+    // camera controller which orbits the plane graphic
+    private val orbitPlaneCameraController by lazy {
+        // instantiate a new camera controller with a distance from airplane graphic
+        OrbitGeoElementCameraController(airplane3DGraphic, 100.0).apply {
+            // set a relative pitch to the target
+            setCameraPitchOffset(3.0)
+            // set a relative heading to the target
+            setCameraHeadingOffset(150.0)
+        }
+    }
+
+    // camera controller for free roam navigation
+    private val globeCameraController = GlobeCameraController()
+
+    // camera looking at the Upheaval Dome crater in Utah
+    private val defaultCamera = Camera(
+        38.459291,
+        -109.937576,
+        5500.0,
+        150.0,
+        20.0,
+        0.0
+    )
+
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        // inflate the camera controller options menu
+        menuInflater.inflate(R.menu.camera_controller_menu, menu)
+        return super.onCreateOptionsMenu(menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
```

Comment:
The menu style approach was helpful for the samples in Runtime. For Maps SDK the drop-down options layout similar to this is replaced `AutoCompleteTextView` where you can set the list of items to an `ArrayAdapter` and get the position `setOnItemClickListener`. Check out Show result of spatial operations [sample](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/tree/main/show-result-of-spatial-operations) for implementation